chain/chain/src/flat_storage_resharder.rs

```diff
@@ -310,6 +321,12 @@ impl FlatStorageResharder {
 
     /// Performs post-processing of shard splitting after all key-values have been moved from parent to
     /// children. `success` indicates whether or not the previous phase was successful.
+    #[tracing::instrument(
+        level = "info",
```

Comment:
is `info` appropriate here given that the instrumentations above are both `debug`?

Reply:
Good point, I've re-thought log levels in this file